assets/src/edit-story/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -132,6 +138,29 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
           </Text>
         </Label>
       </Row>
+      {canMuted && (
```

Comment:
Nit: shouldn't this really be called `canMute` or more accurately `shouldDisplayMuteButton`?